drivers/src/pcr_log.rs

```diff
@@ -71,7 +74,8 @@ pub struct PcrLogEntry {
     // PCR data
     pub pcr_data: [u32; 12],
 
-    pub reserved1: [u8; 4],
+    // PCR Metadata
+    pub metadata: [u8; 4],
```

Comment:
We want to put the SVN field from `StashMeasurementReq` here

Reply:
Actually, thinking more about this. I think we want all of the additional fields from `StashMeasurementReq` in the log:

* metadata (4 bytes)
* svn (4 bytes)
* context (48 bytes)

If we need to expand the log that's ok. Also if you want us to do a follow up PR for the additional log items, that's fine.

Reply:
Let's do a follow-up PR.